extensions/guacamole-auth-sso/modules/guacamole-auth-sso-cas/src/main/java/org/apache/guacamole/auth/cas/conf/ConfigurationService.java

```diff
@@ -26,10 +26,10 @@
 import org.apache.guacamole.GuacamoleException;
 import org.apache.guacamole.GuacamoleServerException;
 import org.apache.guacamole.auth.cas.group.GroupFormat;
-import org.apache.guacamole.environment.Environment;
```

Comment:
This change should probably also be left out if there's no more substantive changes to this file.